docs/current_docs/api/arguments.mdx

```diff
@@ -203,6 +202,12 @@ The result will look like this:
 HELLO, WORLD
 ```
 
+:::note
+When passing optional Boolean arguments (flags):
+- To set the argument to true: `--foo=true` or `--foo` (Dagger CLI), `true` (Dagger Shell)
+- To set the argument to false: `--foo=false` (Dagger CLI), `false` (Dagger Shell)
+:::
```

Comment:
Now the single `true` for shell doesn't apply. It's just when the flag is used, no matter when:
```suggestion
:::note
When passing optional boolean flags:
- To set the argument to true: `--foo=true` or `--foo`
- To set the argument to false: `--foo=false`
:::
```